src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -1347,6 +1347,19 @@ R"(#include <stdio.h>
         }
     }
 
+    void visit_IntegerBitCount(const ASR::IntegerBitCount_t& x) {
+        self().visit_expr(*x.m_a);
+        int arg_kind = ASRUtils::extract_kind_from_ttype_t(x.m_type);
+        switch (arg_kind) {
+            case 1: src = "_lpython_bit_count1(" + src + ")"; break;
+            case 2: src = "_lpython_bit_count2(" + src + ")"; break;
+            case 4: src = "_lpython_bit_count4(" + src + ")"; break;
+            case 8: src = "_lpython_bit_count8(" + src + ")"; break;
+            default: throw CodeGenError("Unsupported Integer Kind: " + \
+                            std::to_string(arg_kind));
```

Comment:
```suggestion
        src = "_lpython_bit_count" + std::to_string(arg_kind) + "(" + src + ")";
```

Reply:
Should I do this for the `visit_IntegerBitLen` function above too?

Reply:
Also, once I do this directly in the ASR, I can remove the entire function right?

Reply:
I meant doing it as an ASR to ASR pass. We should use `BaseExprReplacer`. @certik Do you think https://github.com/lcompilers/lpython/pull/1546#discussion_r1116489030 is a good idea?

Reply:
Yes.